libs/cardano-ledger-core/src/Cardano/Ledger/DRep.hs

```diff
@@ -11,6 +11,7 @@
 module Cardano.Ledger.DRep (
   DRep (DRepCredential, DRepKeyHash, DRepScriptHash, DRepAlwaysAbstain, DRepAlwaysNoConfidence),
   DRepState (..),
+  credToDRep,
```

Comment:
Could you also please mention this addition in [the changelog here](https://github.com/IntersectMBO/cardano-ledger/blob/12e4531da33184cbca080a3ddf7cd0196d516665/libs/cardano-ledger-core/CHANGELOG.md?plain=1#L4) (no need to add a new section, this version has not been released yet)